react/BankInvoiceSection.tsx

```diff
@@ -37,6 +44,8 @@ const BankInvoiceSection: FC = () => {
     message: `"${printInvoice}"`,
   })
 
+  const handles = useCssHandles(CSS_HANDLES)
```

Comment:
Hooks must be called before any kind of forking code. It should be moved to the top of the component. The linter is also catching the error 😉